lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -161,25 +163,32 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       const maxWaste = Math.max(...opportunityAudits.map(audit => audit.result.rawValue));
       const scale = Math.ceil(maxWaste / 1000) * 1000;
       const groupEl = this.renderAuditGroup(groups['load-opportunities'], {expandable: false});
-      opportunityAudits.forEach(item => groupEl.appendChild(this._renderOpportunity(item, scale)));
+      opportunityAudits.forEach((item, i) =>
+          groupEl.appendChild(this._renderOpportunity(item, i, scale)));
       groupEl.open = true;
       element.appendChild(groupEl);
     }
 
     // Diagnostics
     const diagnosticAudits = category.audits
-        .filter(audit => audit.group === 'diagnostics' && audit.result.score < 1);
+        .filter(audit => audit.group === 'diagnostics' && audit.result.score < 1)
+        .sort((a, b) => {
+          const scoreA = a.result.informative ? 100 : a.result.score;
```

Comment:
re:@vinamratasingal's comment on dropdown -- I think once we get the overall left/right alignments on the layout, and the dropdown color being darker as intended, it would look better.  Keeping watching out for any issues though!